receiver/tlscheckreceiver/README.md

```diff
@@ -0,0 +1,37 @@
+# TLS Check Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [contrib] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Ftlscheck%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Ftlscheck) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Ftlscheck%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Ftlscheck) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@michael-burt](https://www.github.com/michael-burt) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+Emit metrics about x.509 certificates.
+
+## Getting Started
+
+By default, the TLS Check Receiver will emit a single metric, `tlscheck.time_left`, per target. This is measured in seconds until the date and time specified in the `NotAfter` field of the x.509 certificate. 
```

Comment:
Is it worth saying explicitly that this value will be negative if the expiry time has been passed?

Reply:
I was thinking it would clamp to 0 after expiry, but open to changing that behavior. I haven't often encountered negative values in gauges. I will document the behavior explicitly.

Reply:
It would allow a user to know how long ago it expired, not just that it had expired.

Reply:
I added it [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35441/files#diff-fd4a88524697321ef8eb94214702d17988629021b4f234aa751899645eefe5dcR19) and [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35441/files#diff-545eea690385fa2a051465cbef7a0487d9f6a9fc47721eef00f97f92487babc5R30).